examples/invoke/invoke-receiver.py

```diff
@@ -0,0 +1,40 @@
+import os
+from concurrent import futures
+import time
+
+import grpc
+import dapr_pb2 as dapr_messages
+import dapr_pb2_grpc as dapr_services
+import daprclient_pb2 as daprclient_messages
+import daprclient_pb2_grpc as daprclient_services
+
+from google.protobuf.any_pb2 import Any
+
+# Start a gRPC client
+port = os.getenv('DAPR_GRPC_PORT')
+channel = grpc.insecure_channel(f"localhost:{port}")
+client = dapr_services.DaprStub(channel)
+print(f"Started gRPC client on DAPR_GRPC_PORT: {port}")
+
+# Our server methods
+class DaprClientServicer(daprclient_services.DaprClientServicer):
+    def OnInvoke(self, request, context):
+        # Return response to caller
```

Comment:
I would add the code here to check if method is `my_method`. if not, I would return error e.g. "unsupported methods"

Reply:
Good suggestion! I'm working on adding this :) I would as well like to add a custom Packing / Unpacking part to it to make the example complete (rather than string encoding everywhere in the Any structure).

Would you by any accident know how to Unpack correctly? Currently I have this for Packing:

```python
def OnInvoke(self, request, context):
    a = Any()
    a.Pack(response_pb2.SampleResponse(action=123456))
    return a
```

And for Unpacking:

```python
response = client.InvokeService(dapr_messages.InvokeServiceEnvelope(id="method-receiver", method="my_method", data=data))
res = response_pb2.SampleResponse()
response.data.Unpack(res)
print(res)
```

But this is somehow not working... would love your thoughts on it :)

Reply:
@youngbupark done, added filter as well as Custom Data Packing / Unpacking